### PR TITLE
Restructure app code

### PR DESCRIPTION
**Pull request by eslong**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -23,10 +23,10 @@ module.exports = {
     rules: {
         'no-var': 2,
         'padded-blocks': 0,
-        'max-len': ['error', 100],
+        'max-len': ['error', 140],
         'comma-dangle': ['error', 'always-multiline'],
         'react/display-name': 2,
-        'react/prefer-stateless-function': 2,
+        'react/prefer-stateless-function': 0,
```
Comment (eatorres) [inline .eslintrc.js]: What's the difference between 2 and 0?

File: .eslintrc.js [hunk 2]
```diff
@@ -23,10 +23,10 @@ module.exports = {
     rules: {
         'no-var': 2,
         'padded-blocks': 0,
-        'max-len': ['error', 100],
+        'max-len': ['error', 140],
```
Comment (eatorres) [inline .eslintrc.js]: Can we leave 120 as an inbetween?

File: .eslintrc.js (hunk 1 above)
Comment (eslong) [reply .eslintrc.js]: 0 disables the warning, 2 only warns on errors

File: .eslintrc.js (hunk 2 above)
Comment (eslong) [reply .eslintrc.js]: Sure, but 140 is the google recommended length

Comment (eatorres) [reply .eslintrc.js]: that's enough to convince me. Let's leave it at 140

**Pull request merged**